### PR TITLE
add new dependencies of spacy-models

### PR DESCRIPTION
**Pull request by h-vetinari**
For https://github.com/conda-forge/spacy-models-feedstock/pull/7

### CODE REVIEW

File: recipes/curated-tokenizers/patches/0001-use-our-own-sentencepiece-abseil.patch [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
++            'absl_flags_config', 'absl_flags_internal', 'absl_flags_marshalling', 'absl_flags_parse',
++            'absl_flags_private_handle_accessor', 'absl_flags_program_name', 'absl_flags_reflection',
++            'absl_flags_usage', 'absl_flags_usage_internal',
++        )
++        EXTENSION_LIBRARIES += ["libprotobuf"]
++        library_path = library_path / "Library"
++    else:
++        absl_glob = pathlib.Path(os.environ['PREFIX']).glob('lib/libabsl_*.so')
++        absl_libs = tuple(lib.stem[3:] for lib in absl_glob)
++        EXTENSION_LIBRARIES += ["protobuf"]
```
Comment (h-vetinari) [inline recipes/curated-tokenizers/patches/0001-use-our-own-sentencepiece-abseil.patch:74]:
```suggestion
+        EXTENSION_LIBRARIES += ["sentencepiece", "protobuf"]
```

**Comment by h-vetinari** [comment]
@conda-forge/help-python 
Newer `spacy-models` versions have new dependencies that aren't in conda-forge yet, specifically:
* `spacy-curated-transformers`, which itself depends on
  * `curated-transformers`
  * `curated-tokenizers`
* `spacy-pkuseg`

The versions are chosen here to satisfy the pins in https://github.com/conda-forge/spacy-models-feedstock/pull/7, now after https://github.com/conda-forge/sentencepiece-feedstock/pull/59 also on windows (there's a spurious failure in conda-build, but that shouldn't make a concern for the content of the PR). Please let me know if you have any comments.

CC @conda-forge/spacy @conda-forge/spacy-models

**Comment by h-vetinari** [comment]
Windows is running into https://github.com/conda/conda-build/issues/5416

**Comment by h-vetinari** [comment]
Remaining failure on windows is due to building a `noarch: python` package and then running into the lack of `pytorch` on windows.

**Review by xhochy — APPROVED** [review]

**Pull request merged**